libraries/AP_GPS/AP_GPS_SBF.cpp

```diff
@@ -49,10 +49,37 @@ do {                                            \
                        OUTOFGEOFENCE)
 
 
+static uint8_t counter = 0;     // For Debuging
+
+
 AP_GPS_SBF::AP_GPS_SBF(AP_GPS &_gps, AP_GPS::GPS_State &_state,
-                       AP_HAL::UARTDriver *_port) :
+                       AP_HAL::UARTDriver *_port, uint8_t _asterx_setup) :
     AP_GPS_Backend(_gps, _state, _port)
 {
+    switch(_asterx_setup) {
+        case 0:
+            _asterx_type_is_singleantenna = true;
+            _asterx_type_is_i = false;
+            _asterx_type_is_dualantenna = false;
+            gcs().send_text(MAV_SEVERITY_INFO, "SBF Single Antenna\n");
```

Comment:
This makes a spammy driver more spammy. Instead just change the `AP_GPS_SBF::name(void)` method to return the appropriate string rather then send an extra status text everywhere.